src/index.js

```diff
@@ -26,7 +26,7 @@
 *
 */
 
-import * as Reducers from './reducers/select';
+import * as Reducers from './reducers/index';
```

Comment:
`index.js` is the default file to use when importing from directory. Replace with `'./reducers';`.